Actualizacion de las tarjetas de restaurante Usando Tailwind

EnriDv opened the pull request:
Se rehicieron los modulos correspondientes a la lista de restaurantes (tarjeta de restaurantes) para que aplique el diseño presentado en figma.
![infig](https://github.com/user-attachments/assets/d6d6c48a-343f-4e7f-951e-0c30f4dd9800)

EnriDv committed:
install tailwind

EnriDv committed:
Creacion del boton para ver restaurantes

EnriDv committed:
crear tarjetas de restaurantes y su redireccion

RaidenMT commented on OrderNow-React/src/components/BotonRestaurantes.jsx:
Después se solicitará modificar estos componentes para hacerlos más transparentes y sean reutilizables

RaidenMT commented on OrderNow-React/src/components/TarjetaRestaurante.jsx:
No me convence el espaciado entre productos, vamos a tener que mirar nuevamente

RaidenMT commented on OrderNow-React/src/components/TarjetaRestaurante.jsx:
Revisar si las fotos serán cuadradas o redondas

RaidenMT commented on OrderNow-React/src/components/TarjetaRestaurante.jsx:
En versión de escritorio no debería ser tan angosto, en móvil está bien
![Screenshot_28-3-2025_12825_ominous-space-pancake-966w6wwrj9jcxwvv-5173 app github dev](https://github.com/user-attachments/assets/987b0cab-e351-4b40-a607-4ea4bac8f7e3)

RaidenMT commented:
Hacer pull antes de subir por favor

EnriDv committed:
Merge branch 'main' into main

EnriDv committed:
change cards

RaidenMT reviewed (commented):
Hacer Pull y avisarme antes de aprobar este PR

EnriDv committed:
Merge branch 'main' into main

EnriDv committed:
Arreglo del App.jsx

Se corrigio el enrutador, cambiando la etiqueta

EnriDv committed:
Merge branch 'master'

RaidenMT commented on OrderNow-React/README.md:
```diff
@@ -3,18 +3,22 @@
 
-    npm install -g npm@latest
-    npm install
-    npm install react-router-dom
```
No borrar esta linea de codigo, es importante

RaidenMT commented on OrderNow-React/src/App.css:
Borrar

EnriDv replied:
listo

EnriDv committed:
Cambios resueltos

Se agrego el comando para instalar react-router-dom
Se borro el archivo App.css
